jetty-servlets/src/main/java/org/eclipse/jetty/servlets/IncludeExcludeBasedFilter.java

```diff
@@ -120,18 +106,6 @@ protected boolean shouldFilter(HttpServletRequest http_request, HttpServletRespo
             return false;
         }
 
-        String content_type = http_response.getContentType();
```

Comment:
This code should be left in, as prior filters may have set the content type, or shouldfilter may be called in other circumstances.

Perhaps if there is no content type available from the response, we should use the requestURI extension to guess a content-type.